x-pack/test/api_integration/deployment_agnostic/services/index.ts

```diff
@@ -17,7 +19,9 @@ export type {
 } from '@kbn/ftr-common-functional-services';
 
 export const services = {
+  ...commonServices,
```

Comment:
Not every service in `'../../../common/services'` is deployment-agnostic and we shouldn't import it like this.
Instead I picked up the ones that work both in serverless/stateful and import as `deploymentAgnosticServices`

If you need any other services from `common/services` let me know which ones so I can review/modify it and load in  https://github.com/elastic/kibana/blob/145ed27ca82e1f2cd52fc230c3d7fed5bdd5d61a/x-pack/test/api_integration/deployment_agnostic/services/deployment_agnostic_services.ts#L15-L28